cost/turbonomics/delete_unattached_volumes/aws/turbonomics_delete_virtual_volumes.pt

```diff
@@ -0,0 +1,369 @@
+name "Turbonomic Delete Unattached Volumes Recommendations AWS"
+rs_pt_ver 20180301
+type "policy"
+short_description "POC turbonomics policy for deleting unattached volumes [README](https://github.com/flexera-public/policy_templates/master/cost/turbonomics/delete_virtual_volume_recommendations) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "0.1",
+  provider: "AWS",
+  source: "Turbonomic",
+  service: "Storage",
+  policy_set: "Unused Volumes",
+  recommendation_type: "Usage Reduction",
+  publish: "false"
+)
+
+##################
+# Parameters   #
+##################
+
+parameter "auth_cookie" do
+  type "string"
+  label "Authorization Cookie"
+  no_echo true
+  description "Valid authorization cookie"
+end
+
+parameter "param_provider" do
+  type "string"
+  label "Cloud Provider"
+  description "Specifies provider where you want to get recommendations"
+  # Once we can add multiple providers, allow: "All", "AWS", "Azure Subscription", "GCP Project"
+  allowed_values "AWS"
+  # Once we can add multiple providers change this default value
+  default "AWS"
+end
+
+parameter "param_days_unattached" do
+  type "number"
+  label "Unused days"
+  description "The number of days a volume has been unused. The days should be greater than zero"
+  default 30
+  min_value 1
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_turbonomic_host" do
+  type "string"
+  label "Turbonomic API Host endpoint"
+  description "Host from your Turbonomic service"
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+#authenticate with password and username - not implemented and needs work done
```

Comment:
@juanestrahcl since this code is not being used I think deleting it would be a good idea.